cli/serve.ts

```diff
@@ -58,12 +59,12 @@ export default async function(...args: string[]) {
         throw error
       },
       async onListen() {
-        console.log(`Capi server listening at "${href}"`)
+        console.log(blue("Created"), "Capi server instance", gray(`at ${href}`))
         await onReady()
       },
     })
   } else {
-    console.log(`Reusing existing Capi server at "${href}"`)
+    console.log(yellow("Reusing"), "Capi server instance", gray(`at ${href}`))
```

Comment:
```suggestion
    console.log(gray("Reusing"), "Capi server instance", gray(`at ${href}`))
```

Reply:
Why gray and not yellow? The yellow correctly indicates some risk (the reused Capi server instance could be stopped midway through the `--`-supplied cmd).